---
layout: page
lang: en
permalink: /software/
title: Software & Data
---

--------

* {% smooth_scroll [\*Sakaue, S., \*Kanai, M. *et al.*, *medRxiv* (2020)](sakaue-s-kanai-m-et-al-medrxiv-2020) %}
* {% smooth_scroll [Kanai, M. *et al.*, *Nat. Genet.* (2018)](kanai-m-et-al-nat-genet-2018) %}
* {% smooth_scroll [Kanai, M. *et al.*, *J. Hum. Genet.* (2016)](kanai-m-et-al-j-hum-genet-2016) %}
* {% smooth_scroll [R packages](r-packages) %}
  * {% smooth_scroll [Grimon (Kanai, M. *et al.*, *Bioinformatics*, 2018)](grimon) %}

--------
## \*Sakaue, S., \*Kanai, M. *et al.*, *medRxiv* (2020)

{% reference Sakaue2020c -f publications.bib %}

* BioBank Japan PheWeb browser is available at [PheWeb.jp](https://pheweb.jp/).
* GWAS summary statistics of 220 complex diseases and traits are available at [the NBDC Human Database](https://humandbs.biosciencedbc.jp/en/hum0197-latest#hum0197.v3.gwas.v1).


## Kanai, M. *et al.*, *Nat. Genet.* (2018)

{% reference Kanai2018a -f publications.bib %}

* GWAS summary statistics of 58 quantitative traits are available at [the NBDC Human Database](https://humandbs.biosciencedbc.jp/hum0014-latest#58qt).

### Analyzed traits
<img src="/img/Kanai2018_study_design.svg" alt="Study design" style="max-width:992px">

--------

<table class="table table-condensed table-striped table-responsive" style="max-width:992px">
<thead>
<tr>
<th>Category</th>
<th>Trait</th>
<th>(Abbr.)</th>
<th style="text-align: right">#samples</th>
</tr>
</thead>
<tbody>
<tr>
<td>Metabolic</td>
<td>Total cholesterol</td>
<td>TC</td>
<td style="text-align: right">128,305</td>
</tr>
<tr>
<td>Metabolic</td>
<td>High density lipoprotein cholesterol</td>
<td>HDL-C</td>
<td style="text-align: right">70,657</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Low density lipoprotein cholesterol</td>
<td>LDL-C</td>
<td style="text-align: right">72,866</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Triglyceride</td>
<td>TG</td>
<td style="text-align: right">105,597</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Blood sugar</td>
<td>BS</td>
<td style="text-align: right">93,146</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Hemoglobin A1c</td>
<td>HbA1c</td>
<td style="text-align: right">42,790</td>
</tr>
<tr>
<td>Protein</td>
<td>Total protein</td>
<td>TP</td>
<td style="text-align: right">113,509</td>
</tr>
<tr>
<td>Protein</td>
<td>Albumin</td>
<td>Alb</td>
<td style="text-align: right">102,223</td>
</tr>
<tr>
<td>Protein</td>
<td>Non-albumin protein</td>
<td>NAP</td>
<td style="text-align: right">98,538</td>
</tr>
<tr>
<td>Protein</td>
<td>Albumin/globulin ratio</td>
<td>A/G</td>
<td style="text-align: right">98,626</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Blood urea nitrogen</td>
<td>BUN</td>
<td style="text-align: right">139,818</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Serum creatinine</td>
<td>sCr</td>
<td style="text-align: right">142,097</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Estimated glomerular filtration rate</td>
<td>eGFR</td>
<td style="text-align: right">143,658</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Uric acid</td>
<td>UA</td>
<td style="text-align: right">109,029</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Sodium</td>
<td>Na</td>
<td style="text-align: right">127,304</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Potassium</td>
<td>K</td>
<td style="text-align: right">132,938</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Chlorine</td>
<td>Cl</td>
<td style="text-align: right">126,402</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Calcium</td>
<td>Ca</td>
<td style="text-align: right">71,701</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Phosphorus</td>
<td>P</td>
<td style="text-align: right">42,793</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Total bilirubin</td>
<td>TBil</td>
<td style="text-align: right">110,207</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Zinc sulfate turbidity test</td>
<td>ZTT</td>
<td style="text-align: right">12,303</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Aspartate aminotransferase</td>
<td>AST</td>
<td style="text-align: right">134,154</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Alanine aminotransferase</td>
<td>ALT</td>
<td style="text-align: right">134,182</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Alkaline phosphatase</td>
<td>ALP</td>
<td style="text-align: right">105,030</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Gamma-glutamyl transferase</td>
<td>GGT</td>
<td style="text-align: right">118,309</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Activated partial thromboplastin time</td>
<td>APTT</td>
<td style="text-align: right">37,767</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Prothrombin time</td>
<td>PT</td>
<td style="text-align: right">58,110</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Fibrinogen</td>
<td>Fbg</td>
<td style="text-align: right">18,348</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Creatine kinase</td>
<td>CK</td>
<td style="text-align: right">106,080</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Lactate dehydrogenase</td>
<td>LDH</td>
<td style="text-align: right">126,319</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>C-reactive protein</td>
<td>CRP</td>
<td style="text-align: right">75,391</td>
</tr>
<tr>
<td>Hematological</td>
<td>White blood cell count</td>
<td>WBC</td>
<td style="text-align: right">107,964</td>
</tr>
<tr>
<td>Hematological</td>
<td>Neutrophil count</td>
<td>Neutro</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Eosinophil count</td>
<td>Eosino</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Basophil count</td>
<td>Baso</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Monocyte count</td>
<td>Mono</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Lymphocyte count</td>
<td>Lym</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Red blood cell count</td>
<td>RBC</td>
<td style="text-align: right">108,794</td>
</tr>
<tr>
<td>Hematological</td>
<td>Hemoglobin</td>
<td>Hb</td>
<td style="text-align: right">108,769</td>
</tr>
<tr>
<td>Hematological</td>
<td>Hematocrit</td>
<td>Ht</td>
<td style="text-align: right">108,757</td>
</tr>
<tr>
<td>Hematological</td>
<td>Mean corpuscular volume</td>
<td>MCV</td>
<td style="text-align: right">108,256</td>
</tr>
<tr>
<td>Hematological</td>
<td>Mean corpuscular hemoglobin</td>
<td>MCH</td>
<td style="text-align: right">108,054</td>
</tr>
<tr>
<td>Hematological</td>
<td>Mean corpuscular hemoglobin concentration</td>
<td>MCHC</td>
<td style="text-align: right">108,728</td>
</tr>
<tr>
<td>Hematological</td>
<td>Platelet count</td>
<td>Plt</td>
<td style="text-align: right">108,208</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Systolic blood pressure</td>
<td>SBP</td>
<td style="text-align: right">136,597</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Diastolic blood pressure</td>
<td>DBP</td>
<td style="text-align: right">136,615</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Mean arterial pressure</td>
<td>MAP</td>
<td style="text-align: right">136,482</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Pulse pressure</td>
<td>PP</td>
<td style="text-align: right">136,249</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Interventricular septum thickness</td>
<td>IVS</td>
<td style="text-align: right">19,318</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Posterior wall thickness</td>
<td>PW</td>
<td style="text-align: right">19,373</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular internal dimension in diastole</td>
<td>LVDd</td>
<td style="text-align: right">19,676</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular internal dimension in systole</td>
<td>LVDs</td>
<td style="text-align: right">19,586</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular mass</td>
<td>LVM</td>
<td style="text-align: right">19,076</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular mass index</td>
<td>LVMI</td>
<td style="text-align: right">17,837</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Relative wall thickness</td>
<td>RWT</td>
<td style="text-align: right">19,311</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Fractional shortening</td>
<td>FS</td>
<td style="text-align: right">19,580</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Ejection fraction</td>
<td>EF</td>
<td style="text-align: right">19,516</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>E/A ratio</td>
<td>E/A</td>
<td style="text-align: right">8,600</td>
</tr>
<tfoot>
<tr>
<td colspan="4"><sup>*</sup> Detailed chracteristics of these traits were available in Supplementary Table 1 of the paper.</td>
</tr>
</tfoot>
</tbody></table>

### Fuji plot (Figure 1) {% github_link fujiplot %}
[Fuji plot](https://github.com/mkanai/fujiplot) is a circos representation of multiple GWAS reseluts, which was designed to efficiently summarize significant SNPs across dozens of traits. We provide a wrapper script for [Circos](http://circos.ca/), generating input data and configuration files from GWAS data. 

<img src="/img/Kanai2018_Fig1.svg" alt="Fuji plot" width="100%" style="max-width:768px">

### Genetic correlation corrplot (Figure 2) {% github_link ldsc-corrplot-rg %}
We modified [corrplot](https://github.com/taiyun/corrplot) to visualize pairwise genetic correlations estimated via [bivariate LD score regression](https://www.nature.com/articles/ng.3406).

<img src="/img/Kanai2018_Fig2.svg" alt="Genetic correlation corrplot" width="100%" style="max-width:768px">

### Genetic correlation network (Figure 3) {% github_link ldsc-network-plot %}
Genetic correlation network was designed to visualize cross-phenotype relationships (pairwise genetic correlation matrix) across dozens of traits estimated via [bivariate LD score regression](https://www.nature.com/articles/ng.3406).

<img src="/img/Kanai2018_Fig3.svg" alt="Genetic correlation network" width="100%" style="max-width:768px">

### Cell-type specificity network (Figure 6) {% github_link ldsc-network-plot %}
Cell-type specificity network was designed to summarize cell-type specific enrichments estimated via [stratified LD score regression](https://www.nature.com/articles/ng.3404).

<img src="/img/Kanai2018_Fig6.svg" alt="Genetic correlation network" width="100%" style="max-width:768px">


### Links
* [the NBDC Human Database](https://humandbs.biosciencedbc.jp/hum0014-latest#58qt)
* [The Biobank Japan Project](https://biobankjp.org/english/)
* [PLINK 1.9](https://www.cog-genomics.org/plink2)
* [ldsc](https://github.com/bulik/ldsc)
* [LD score](https://data.broadinstitute.org/alkesgroup/LDSCORE/) (Price Lab website)
  * [Additional LD score for East Asians](http://jenger.riken.jp/en/data) (our Lab website)
* [Fuji plot](https://github.com/mkanai/fujiplot)
* [ldsc-corrplot-rg](https://github.com/mkanai/ldsc-corrplot-rg)
* [ldsc-network-plot](https://github.com/mkanai/ldsc-network-plot)
* [circos](http://circos.ca/)
* [corrplot](https://github.com/taiyun/corrplot)
* [1000 Genomes Project](http://www.1000genomes.org/)

--------

## Kanai, M. *et al.*, *J. Hum. Genet.* (2016)

{% reference Kanai2016 -f publications.bib %}

* The authorsâ€™ scripts used for calculations: {% github_link GWAS_threshold_scripts %}
* {% smooth_scroll [WRShd](wrshd) %}
* [PLINK 1.9](https://www.cog-genomics.org/plink2)
* [EIGENSOFT 6.0.1](http://www.hsph.harvard.edu/alkes-price/software/)
* [1000 Genomes Project](http://www.1000genomes.org/)

--------

## R packages

* {% smooth_scroll [Grimon](grimon) %}
* {% smooth_scroll [deFinetti](definetti) %}
* {% smooth_scroll [ccmatch](ccmatch) %}
* {% smooth_scroll [WRShd](wrshd) %}


### Installation
You can install any of my packages via the following code in R:

```r
if (!require("devtools")) {
  install.packages("devtools")
}
devtools::install_github("mkanai/{package_name}")
```

--------
<h3 id="grimon"> Grimon (Kanai, M. <em>et al.</em>, <em>Bioinformatics</em>, 2018)</h3>
{% github_link grimon %}

{% reference Kanai2018b -f publications.bib %}

<p><img src="http://mkanai.github.io/img/Grimon_logo.png" width="360px"></p>

Grimon (**Gr**aphical **i**nterface to visualize **m**ulti-**o**mics **n**etworks) visualizes high-dimensional multi-layered data sets in three-dimensional parallel coordinates. It enables users to intuitively and interactively explore their data, helping their understanding of multiple inter-layer connections embedded in high-dimensional complex data.

#### Schematic overview
<p><img src="http://mkanai.github.io/img/Grimon_demo.png" width=768px></p>


#### Example 1: Metagenome data {% cite Maeda2016 -f references.bib %}
<p><img src="http://mkanai.github.io/img/Grimon_example1.png" width=768px></p>


#### Example 2: the Geuvadis Project {% cite Lappalainen2013 -f references.bib %}
<p><img src="http://mkanai.github.io/img/Grimon_example2.png" width=768px></p>


**References**

1. {% reference Maeda2016 -f references.bib %}
2. {% reference Lappalainen2013 -f references.bib %}


--------
### deFinetti
{% github_link deFinetti! %}

A package for plotting a [de Finetti diagram](http://en.wikipedia.org/wiki/De_Finetti_diagram) and distributions of [F-statistics](http://en.wikipedia.org/wiki/F-statistics) of genotypes.

![deFinetti_example.png](/img/deFinetti_example.png)

--------
### ccmatch
{% github_link ccmatch! %}

A package for optimal matching of cases to controls using network flow theory.

**Reference**

* {% reference Rosenbaum1989 -f references.bib %}


--------
### WRShd
{% github_link WRShd! %}

This package provides C++ sub-routines for the Harrell-Davis estimate functions from R. R. Wilcox' robust statistics package [`WRS`](https://github.com/nicebread/WRS), which substantially speed up the original implementation in R.


[[back]]({{ site.url }})

