<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Software & Data</title>
  <meta name="description" content="">

  <!-- for iPhone/iPad Safari -->
  <meta name="format-detection" content="telephone=no">
  
  <!-- Loading Bootstrap -->
  <link rel="stylesheet" href="/css/vendor/bootstrap/css/bootstrap.min.css">

  <!-- Loading Flat UI -->
  <link rel="stylesheet" href="/css/flat-ui.min.css">
  
  <link rel="stylesheet" href="/css/syntax.css">
  
  <link rel="stylesheet" href="/css/custom.css">

  

  <!-- HTML5 shim, for IE6-8 support of HTML5 elements. All other JS at the end of file. -->
  <!--[if lt IE 9]>
    <script src="js/vendor/html5shiv.js"></script>
    <script src="js/vendor/respond.min.js"></script>
  <![endif]-->

  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-BFKLBV2ZNN"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-BFKLBV2ZNN');
  </script>

  <link rel="canonical" href="https://emanuelecannizzaro.github.io/software/">
  <link rel="alternate" type="application/rss+xml" title="" href="https://emanuelecannizzaro.github.io/feed.xml" />
</head>


  <body>
    <div id="wrap">
          <nav class="navbar navbar-inverse navbar-embossed" role="navigation">
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header">
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-collapse-8">
          <span class="sr-only">Toggle navigation</span>
        </button>
        <a class="navbar-brand" href="https://emanuelecannizzaro.github.io">Emanuele Cannizzaro</a>
      </div>

      <!-- Collect the nav links, forms, and other content for toggling -->
      <div class="collapse navbar-collapse" id="navbar-collapse-8">
        
        <ul class="nav navbar-nav navbar-right">
          
          
          <li><a href="https://emanuelecannizzaro.github.io/ja/">日本語</a></li>
          
          <li><a class="navbar-text" rel="noopener" target="_blank" href="https:/github.com/emanuelecannizzaro/emanuelecannizzaro.github.io/commits/master">Last updated: 2022-08-09</a></li>
        </ul>
      </div><!-- /.navbar-collapse -->
    </nav>


      <div class="container-fluid">
        <div class="row-fluid">
              <header>
      <h1>Software & Data</h1>
    </header>
    <article>
      <hr />

<ul>
  <li><a data-scroll="" href="#sakaue-s-kanai-m-et-al-medrxiv-2020">*Sakaue, S., *Kanai, M. <em>et al.</em>, <em>medRxiv</em> (2020)</a></li>
  <li><a data-scroll="" href="#kanai-m-et-al-nat-genet-2018">Kanai, M. <em>et al.</em>, <em>Nat. Genet.</em> (2018)</a></li>
  <li><a data-scroll="" href="#kanai-m-et-al-j-hum-genet-2016">Kanai, M. <em>et al.</em>, <em>J. Hum. Genet.</em> (2016)</a></li>
  <li><a data-scroll="" href="#r-packages">R packages</a>
    <ul>
      <li><a data-scroll="" href="#grimon">Grimon (Kanai, M. <em>et al.</em>, <em>Bioinformatics</em>, 2018)</a></li>
    </ul>
  </li>
</ul>

<hr />
<h2 id="sakaue-s-kanai-m-et-al-medrxiv-2020">*Sakaue, S., *Kanai, M. <em>et al.</em>, <em>medRxiv</em> (2020)</h2>

<p><span id="Sakaue2020c">*Sakaue, S., <strong>*Kanai, M</strong>., Tanigawa, Y., Karjalainen, J., Kurki, M., Koshiba, S. <i>et al.</i> <a href="http://dx.doi.org/10.1101/2020.10.23.20213652" target="_blank" rel="noopener">A global atlas of genetic associations of 220 deep phenotypes.</a> <i class="journal">medRxiv</i> (2020). doi:10.1101/2020.10.23.20213652</span></p>

<ul>
  <li>BioBank Japan PheWeb browser is available at <a href="https://pheweb.jp/">PheWeb.jp</a>.</li>
  <li>GWAS summary statistics of 220 complex diseases and traits are available at <a href="https://humandbs.biosciencedbc.jp/en/hum0197-latest#hum0197.v3.gwas.v1">the NBDC Human Database</a>.</li>
</ul>

<h2 id="kanai-m-et-al-nat-genet-2018">Kanai, M. <em>et al.</em>, <em>Nat. Genet.</em> (2018)</h2>

<p><span id="Kanai2018a"><strong>Kanai, M</strong>., Akiyama, M., Takahashi, A., Matoba, N., Momozawa, Y., Ikeda, M. <i>et al.</i> <a href="http://dx.doi.org/10.1038/s41588-018-0047-6" target="_blank" rel="noopener">Genetic analysis of quantitative traits in the Japanese population links cell types to complex human diseases.</a> <i class="journal">Nature Genetics</i> <b>50,</b> 390–400 (2018).  <a class="btn btn-primary btn-xs fui-link" href="https://www.ncbi.nlm.nih.gov/pubmed/29403010" target="_blank" rel="noopener"><span class="icon-text">PubMed</span></a> <a class="btn btn-primary btn-xs fui-link" href="http://www.riken.jp/pr/press/2018/20180206_1/" target="_blank" rel="noopener"><span class="icon-text">PR</span></a> <a class="btn btn-primary btn-xs fui-link" href="http://www.riken.jp/en/research/rikenresearch/highlights/20180427_FY20180002/" target="_blank" rel="noopener"><span class="icon-text">RIKEN Research Highlight</span></a></span></p>

<ul>
  <li>GWAS summary statistics of 58 quantitative traits are available at <a href="https://humandbs.biosciencedbc.jp/hum0014-latest#58qt">the NBDC Human Database</a>.</li>
</ul>

<h3 id="analyzed-traits">Analyzed traits</h3>
<p><img src="/img/Kanai2018_study_design.svg" alt="Study design" style="max-width:992px" /></p>

<hr />

<table class="table table-condensed table-striped table-responsive" style="max-width:992px">
<thead>
<tr>
<th>Category</th>
<th>Trait</th>
<th>(Abbr.)</th>
<th style="text-align: right">#samples</th>
</tr>
</thead>
<tbody>
<tr>
<td>Metabolic</td>
<td>Total cholesterol</td>
<td>TC</td>
<td style="text-align: right">128,305</td>
</tr>
<tr>
<td>Metabolic</td>
<td>High density lipoprotein cholesterol</td>
<td>HDL-C</td>
<td style="text-align: right">70,657</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Low density lipoprotein cholesterol</td>
<td>LDL-C</td>
<td style="text-align: right">72,866</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Triglyceride</td>
<td>TG</td>
<td style="text-align: right">105,597</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Blood sugar</td>
<td>BS</td>
<td style="text-align: right">93,146</td>
</tr>
<tr>
<td>Metabolic</td>
<td>Hemoglobin A1c</td>
<td>HbA1c</td>
<td style="text-align: right">42,790</td>
</tr>
<tr>
<td>Protein</td>
<td>Total protein</td>
<td>TP</td>
<td style="text-align: right">113,509</td>
</tr>
<tr>
<td>Protein</td>
<td>Albumin</td>
<td>Alb</td>
<td style="text-align: right">102,223</td>
</tr>
<tr>
<td>Protein</td>
<td>Non-albumin protein</td>
<td>NAP</td>
<td style="text-align: right">98,538</td>
</tr>
<tr>
<td>Protein</td>
<td>Albumin/globulin ratio</td>
<td>A/G</td>
<td style="text-align: right">98,626</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Blood urea nitrogen</td>
<td>BUN</td>
<td style="text-align: right">139,818</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Serum creatinine</td>
<td>sCr</td>
<td style="text-align: right">142,097</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Estimated glomerular filtration rate</td>
<td>eGFR</td>
<td style="text-align: right">143,658</td>
</tr>
<tr>
<td>Kidney-related</td>
<td>Uric acid</td>
<td>UA</td>
<td style="text-align: right">109,029</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Sodium</td>
<td>Na</td>
<td style="text-align: right">127,304</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Potassium</td>
<td>K</td>
<td style="text-align: right">132,938</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Chlorine</td>
<td>Cl</td>
<td style="text-align: right">126,402</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Calcium</td>
<td>Ca</td>
<td style="text-align: right">71,701</td>
</tr>
<tr>
<td>Electrolyte</td>
<td>Phosphorus</td>
<td>P</td>
<td style="text-align: right">42,793</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Total bilirubin</td>
<td>TBil</td>
<td style="text-align: right">110,207</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Zinc sulfate turbidity test</td>
<td>ZTT</td>
<td style="text-align: right">12,303</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Aspartate aminotransferase</td>
<td>AST</td>
<td style="text-align: right">134,154</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Alanine aminotransferase</td>
<td>ALT</td>
<td style="text-align: right">134,182</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Alkaline phosphatase</td>
<td>ALP</td>
<td style="text-align: right">105,030</td>
</tr>
<tr>
<td>Liver-related</td>
<td>Gamma-glutamyl transferase</td>
<td>GGT</td>
<td style="text-align: right">118,309</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Activated partial thromboplastin time</td>
<td>APTT</td>
<td style="text-align: right">37,767</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Prothrombin time</td>
<td>PT</td>
<td style="text-align: right">58,110</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Fibrinogen</td>
<td>Fbg</td>
<td style="text-align: right">18,348</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Creatine kinase</td>
<td>CK</td>
<td style="text-align: right">106,080</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>Lactate dehydrogenase</td>
<td>LDH</td>
<td style="text-align: right">126,319</td>
</tr>
<tr>
<td>Other biochemical</td>
<td>C-reactive protein</td>
<td>CRP</td>
<td style="text-align: right">75,391</td>
</tr>
<tr>
<td>Hematological</td>
<td>White blood cell count</td>
<td>WBC</td>
<td style="text-align: right">107,964</td>
</tr>
<tr>
<td>Hematological</td>
<td>Neutrophil count</td>
<td>Neutro</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Eosinophil count</td>
<td>Eosino</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Basophil count</td>
<td>Baso</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Monocyte count</td>
<td>Mono</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Lymphocyte count</td>
<td>Lym</td>
<td style="text-align: right">62,076</td>
</tr>
<tr>
<td>Hematological</td>
<td>Red blood cell count</td>
<td>RBC</td>
<td style="text-align: right">108,794</td>
</tr>
<tr>
<td>Hematological</td>
<td>Hemoglobin</td>
<td>Hb</td>
<td style="text-align: right">108,769</td>
</tr>
<tr>
<td>Hematological</td>
<td>Hematocrit</td>
<td>Ht</td>
<td style="text-align: right">108,757</td>
</tr>
<tr>
<td>Hematological</td>
<td>Mean corpuscular volume</td>
<td>MCV</td>
<td style="text-align: right">108,256</td>
</tr>
<tr>
<td>Hematological</td>
<td>Mean corpuscular hemoglobin</td>
<td>MCH</td>
<td style="text-align: right">108,054</td>
</tr>
<tr>
<td>Hematological</td>
<td>Mean corpuscular hemoglobin concentration</td>
<td>MCHC</td>
<td style="text-align: right">108,728</td>
</tr>
<tr>
<td>Hematological</td>
<td>Platelet count</td>
<td>Plt</td>
<td style="text-align: right">108,208</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Systolic blood pressure</td>
<td>SBP</td>
<td style="text-align: right">136,597</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Diastolic blood pressure</td>
<td>DBP</td>
<td style="text-align: right">136,615</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Mean arterial pressure</td>
<td>MAP</td>
<td style="text-align: right">136,482</td>
</tr>
<tr>
<td>Blood pressure</td>
<td>Pulse pressure</td>
<td>PP</td>
<td style="text-align: right">136,249</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Interventricular septum thickness</td>
<td>IVS</td>
<td style="text-align: right">19,318</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Posterior wall thickness</td>
<td>PW</td>
<td style="text-align: right">19,373</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular internal dimension in diastole</td>
<td>LVDd</td>
<td style="text-align: right">19,676</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular internal dimension in systole</td>
<td>LVDs</td>
<td style="text-align: right">19,586</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular mass</td>
<td>LVM</td>
<td style="text-align: right">19,076</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Left ventricular mass index</td>
<td>LVMI</td>
<td style="text-align: right">17,837</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Relative wall thickness</td>
<td>RWT</td>
<td style="text-align: right">19,311</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Fractional shortening</td>
<td>FS</td>
<td style="text-align: right">19,580</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>Ejection fraction</td>
<td>EF</td>
<td style="text-align: right">19,516</td>
</tr>
<tr>
<td>Echocardiographic</td>
<td>E/A ratio</td>
<td>E/A</td>
<td style="text-align: right">8,600</td>
</tr>
<tfoot>
<tr>
<td colspan="4"><sup>*</sup> Detailed chracteristics of these traits were available in Supplementary Table 1 of the paper.</td>
</tr>
</tfoot>
</tbody></table>

<h3 id="fuji-plot-figure-1-github">Fuji plot (Figure 1) <a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/fujiplot" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a></h3>
<p><a href="https://github.com/mkanai/fujiplot">Fuji plot</a> is a circos representation of multiple GWAS reseluts, which was designed to efficiently summarize significant SNPs across dozens of traits. We provide a wrapper script for <a href="http://circos.ca/">Circos</a>, generating input data and configuration files from GWAS data.</p>

<p><img src="/img/Kanai2018_Fig1.svg" alt="Fuji plot" width="100%" style="max-width:768px" /></p>

<h3 id="genetic-correlation-corrplot-figure-2-github">Genetic correlation corrplot (Figure 2) <a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/ldsc-corrplot-rg" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a></h3>
<p>We modified <a href="https://github.com/taiyun/corrplot">corrplot</a> to visualize pairwise genetic correlations estimated via <a href="https://www.nature.com/articles/ng.3406">bivariate LD score regression</a>.</p>

<p><img src="/img/Kanai2018_Fig2.svg" alt="Genetic correlation corrplot" width="100%" style="max-width:768px" /></p>

<h3 id="genetic-correlation-network-figure-3-github">Genetic correlation network (Figure 3) <a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/ldsc-network-plot" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a></h3>
<p>Genetic correlation network was designed to visualize cross-phenotype relationships (pairwise genetic correlation matrix) across dozens of traits estimated via <a href="https://www.nature.com/articles/ng.3406">bivariate LD score regression</a>.</p>

<p><img src="/img/Kanai2018_Fig3.svg" alt="Genetic correlation network" width="100%" style="max-width:768px" /></p>

<h3 id="cell-type-specificity-network-figure-6-github">Cell-type specificity network (Figure 6) <a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/ldsc-network-plot" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a></h3>
<p>Cell-type specificity network was designed to summarize cell-type specific enrichments estimated via <a href="https://www.nature.com/articles/ng.3404">stratified LD score regression</a>.</p>

<p><img src="/img/Kanai2018_Fig6.svg" alt="Genetic correlation network" width="100%" style="max-width:768px" /></p>

<h3 id="links">Links</h3>
<ul>
  <li><a href="https://humandbs.biosciencedbc.jp/hum0014-latest#58qt">the NBDC Human Database</a></li>
  <li><a href="https://biobankjp.org/english/">The Biobank Japan Project</a></li>
  <li><a href="https://www.cog-genomics.org/plink2">PLINK 1.9</a></li>
  <li><a href="https://github.com/bulik/ldsc">ldsc</a></li>
  <li><a href="https://data.broadinstitute.org/alkesgroup/LDSCORE/">LD score</a> (Price Lab website)
    <ul>
      <li><a href="http://jenger.riken.jp/en/data">Additional LD score for East Asians</a> (our Lab website)</li>
    </ul>
  </li>
  <li><a href="https://github.com/mkanai/fujiplot">Fuji plot</a></li>
  <li><a href="https://github.com/mkanai/ldsc-corrplot-rg">ldsc-corrplot-rg</a></li>
  <li><a href="https://github.com/mkanai/ldsc-network-plot">ldsc-network-plot</a></li>
  <li><a href="http://circos.ca/">circos</a></li>
  <li><a href="https://github.com/taiyun/corrplot">corrplot</a></li>
  <li><a href="http://www.1000genomes.org/">1000 Genomes Project</a></li>
</ul>

<hr />

<h2 id="kanai-m-et-al-j-hum-genet-2016">Kanai, M. <em>et al.</em>, <em>J. Hum. Genet.</em> (2016)</h2>

<p><span id="Kanai2016"><strong>Kanai, M</strong>., Tanaka, T. &amp; Okada, Y. <a href="http://dx.doi.org/10.1038/jhg.2016.72" target="_blank" rel="noopener">Empirical estimation of genome-wide significance thresholds based on the 1000 Genomes Project data set.</a> <i class="journal">Journal of Human Genetics</i> <b>61,</b> 861–866 (2016).  <a class="btn btn-primary btn-xs fui-link" href="http://www.ncbi.nlm.nih.gov/pubmed/27305981" target="_blank" rel="noopener"><span class="icon-text">PubMed</span></a></span></p>

<ul>
  <li>The authors’ scripts used for calculations: <a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/GWAS_threshold_scripts" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a></li>
  <li><a data-scroll="" href="#wrshd">WRShd</a></li>
  <li><a href="https://www.cog-genomics.org/plink2">PLINK 1.9</a></li>
  <li><a href="http://www.hsph.harvard.edu/alkes-price/software/">EIGENSOFT 6.0.1</a></li>
  <li><a href="http://www.1000genomes.org/">1000 Genomes Project</a></li>
</ul>

<hr />

<h2 id="r-packages">R packages</h2>

<ul>
  <li><a data-scroll="" href="#grimon">Grimon</a></li>
  <li><a data-scroll="" href="#definetti">deFinetti</a></li>
  <li><a data-scroll="" href="#ccmatch">ccmatch</a></li>
  <li><a data-scroll="" href="#wrshd">WRShd</a></li>
</ul>

<h3 id="installation">Installation</h3>
<p>You can install any of my packages via the following code in R:</p>

<div class="language-r highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="k">if</span><span class="w"> </span><span class="p">(</span><span class="o">!</span><span class="n">require</span><span class="p">(</span><span class="s2">"devtools"</span><span class="p">))</span><span class="w"> </span><span class="p">{</span><span class="w">
  </span><span class="n">install.packages</span><span class="p">(</span><span class="s2">"devtools"</span><span class="p">)</span><span class="w">
</span><span class="p">}</span><span class="w">
</span><span class="n">devtools</span><span class="o">::</span><span class="n">install_github</span><span class="p">(</span><span class="s2">"mkanai/{package_name}"</span><span class="p">)</span><span class="w">
</span></code></pre></div></div>

<hr />
<h3 id="grimon"> Grimon (Kanai, M. <em>et al.</em>, <em>Bioinformatics</em>, 2018)</h3>
<p><a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/grimon" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a></p>

<p><span id="Kanai2018b"><strong>Kanai, M</strong>., Maeda, Y. &amp; Okada, Y. <a href="http://dx.doi.org/10.1093/bioinformatics/bty488" target="_blank" rel="noopener">Grimon: graphical interface to visualize multi-omics networks.</a> <i class="journal">Bioinformatics</i> <b>34,</b> 3934–3936 (2018).  <a class="btn btn-primary btn-xs fui-link" href="https://www.ncbi.nlm.nih.gov/pubmed/29931190" target="_blank" rel="noopener"><span class="icon-text">PubMed</span></a></span></p>

<p><img src="http://mkanai.github.io/img/Grimon_logo.png" width="360px" /></p>

<p>Grimon (<strong>Gr</strong>aphical <strong>i</strong>nterface to visualize <strong>m</strong>ulti-<strong>o</strong>mics <strong>n</strong>etworks) visualizes high-dimensional multi-layered data sets in three-dimensional parallel coordinates. It enables users to intuitively and interactively explore their data, helping their understanding of multiple inter-layer connections embedded in high-dimensional complex data.</p>

<h4 id="schematic-overview">Schematic overview</h4>
<p><img src="http://mkanai.github.io/img/Grimon_demo.png" width="768px" /></p>

<h4 id="example-1-metagenome-data-1">Example 1: Metagenome data <a class="citation" href="#Maeda2016"><span style="vertical-align: super">1</span></a></h4>
<p><img src="http://mkanai.github.io/img/Grimon_example1.png" width="768px" /></p>

<h4 id="example-2-the-geuvadis-project-2">Example 2: the Geuvadis Project <a class="citation" href="#Lappalainen2013"><span style="vertical-align: super">2</span></a></h4>
<p><img src="http://mkanai.github.io/img/Grimon_example2.png" width="768px" /></p>

<p><strong>References</strong></p>

<ol>
  <li><span id="Maeda2016">Maeda, Y., Kurakawa, T., Umemoto, E., Motooka, D., Ito, Y., Gotoh, K. <i>et al.</i> <a href="http://dx.doi.org/10.1002/art.39783" target="_blank" rel="noopener">Dysbiosis Contributes to Arthritis Development via Activation of Autoreactive T Cells in the Intestine.</a> <i class="journal">Arthritis &amp; Rheumatology</i> <b>68,</b> 2646–2661 (2016). </span></li>
  <li><span id="Lappalainen2013">Lappalainen, T., Sammeth, M., Friedländer, M. R., ‘t Hoen, P. A. C., Monlong, J., Rivas, M. A. <i>et al.</i> <a href="http://dx.doi.org/10.1038/nature12531" target="_blank" rel="noopener">Transcriptome and genome sequencing uncovers functional variation in humans.</a> <i class="journal">Nature</i> <b>501,</b> 506–511 (2013). </span></li>
</ol>

<hr />
<h3 id="definetti">deFinetti</h3>
<p><a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/deFinetti" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a> <a href="https://travis-ci.org/mkanai/deFinetti" target="_blank" rel="noopener"><img src="https://travis-ci.org/mkanai/deFinetti.svg?branch=master" alt="Build Status" /></a></p>

<p>A package for plotting a <a href="http://en.wikipedia.org/wiki/De_Finetti_diagram">de Finetti diagram</a> and distributions of <a href="http://en.wikipedia.org/wiki/F-statistics">F-statistics</a> of genotypes.</p>

<p><img src="/img/deFinetti_example.png" alt="deFinetti_example.png" /></p>

<hr />
<h3 id="ccmatch">ccmatch</h3>
<p><a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/ccmatch" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a> <a href="https://travis-ci.org/mkanai/ccmatch" target="_blank" rel="noopener"><img src="https://travis-ci.org/mkanai/ccmatch.svg?branch=master" alt="Build Status" /></a></p>

<p>A package for optimal matching of cases to controls using network flow theory.</p>

<p><strong>Reference</strong></p>

<ul>
  <li><span id="Rosenbaum1989">Rosenbaum, P. R. <a href="http://dx.doi.org/10.1080/01621459.1989.10478868" target="_blank" rel="noopener">Optimal matching for observational studies.</a> <i class="journal">Journal of the American Statistical Association</i> <b>84,</b> 1024–1032 (1989). </span></li>
</ul>

<hr />
<h3 id="wrshd">WRShd</h3>
<p><a class="btn btn-primary btn-xs fui-github" href="https://github.com/mkanai/WRShd" target="_blank" rel="noopener"><span class="icon-text">GitHub</span></a> <a href="https://travis-ci.org/mkanai/WRShd" target="_blank" rel="noopener"><img src="https://travis-ci.org/mkanai/WRShd.svg?branch=master" alt="Build Status" /></a></p>

<p>This package provides C++ sub-routines for the Harrell-Davis estimate functions from R. R. Wilcox’ robust statistics package <a href="https://github.com/nicebread/WRS"><code class="language-plaintext highlighter-rouge">WRS</code></a>, which substantially speed up the original implementation in R.</p>

<p><a href="https://emanuelecannizzaro.github.io">[back]</a></p>


    </article>


        </div>
      </div>
    </div>

    <!-- jQuery (necessary for Flat UI's JavaScript plugins) -->
    <script src="/js/vendor/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="/js/vendor/video.js"></script>
    <script src="/js/flat-ui.min.js"></script>
    
    <!-- smooth-scroll -->
    <script src="/js/vendor/smooth-scroll.min.js"></script>
    <script>
      smoothScroll.init();
    </script>

        <footer>
      <div class="container">
        <div class="row text-center">
          <ul class="list-inline">
            <li><p><strong>Emanuele Cannizzaro</strong></p></li>
            <li class="hide-small"><span class="fui-mail"></span> <a>Emanuele.Cannizzaro@gmail.com</a></li>
            <li class="hide-small"><span class="fui-twitter"></span> <a href="https://twitter.com/" target="_blank">  @ </a></li>
            <li class="hide-small"><span class="fui-github"></span> <a href="https://github.com/" target="_blank">  @ </a></li>
          </ul>
        </div>
      </div>
    </footer>


  </body>

</html>
